tensorboard/webapp/app_routing/route_registry_module.ts

```diff
@@ -48,7 +48,7 @@ export class RouteRegistryModule {
     }
     this.routeConfigs = new RouteConfigs(configs);
     configs.forEach((config) => {
-      if (config.routeKind) {
+      if (config.routeKind !== null) {
```

Comment:
Could you kindly add a test case for testing this?

Reply:
Thanks for the quick review! Added a test case at: https://github.com/tensorflow/tensorboard/pull/4933/files#diff-cb6149a2dba8807661d981d97d6c471cae461bbe915898cb2ecd48efd68c8dfa

Reply:
Ah, indeed. Sorry about misreading it.